### PR TITLE
Publish to NPM and add this as a CLI

### PR DESCRIPTION
**Pull request by reflexdemon**
This is in very early stage and I have tried to make this run like a CLI so that we can just do `npm install -g mvn-search` and this will add `mvn-search` to command-line.

If you are merging this request, you will have to also plan to publish to NPM repository. Verify the details on `package.json` before publishing it to npm repository. and let me know if you need any help in publishing.

Demo using `npm link` is provided here [Demo Link](https://terminalizer.com/view/6376b1f14184)

This is an effort to address #2

### CODE REVIEW

**Comment by reflexdemon** [comment]
A more detailed demo of `maven-search` as a command line.

[Click to view demo](https://terminalizer.com/view/0dee661e4185)

File: src/cli.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const maven = requir('./index');
```
Comment (chillinPanda) [inline src/cli.js:1]: Hi @reflexdemon,

I think there is a typo `requir` is missing an `e` in the end.

Comment (reflexdemon) [reply src/cli.js:1]: That is a good catch. I will revisit and update shortly

Comment (reflexdemon) [reply src/cli.js]: Actually that file was not required. It is never used inside the application and I have removed the file now. Thanks for bring this to my attention.

**Comment by erosb** [comment]
Published, now it can be installed with `npm i -g @erosb/mvn-search`.
Thanks again for the suggestion.

**Pull request merged**